Core/include/Acts/Utilities/Helpers.hpp

```diff
@@ -187,9 +173,9 @@ std::array<typename T::value_type, 2u> min_max(const T& tseries) {
 /// @return [ range, medium ] in an tuple
 template <typename T>
 std::tuple<typename T::value_type, ActsScalar> range_medium(const T& tseries) {
-  auto [min, max] = min_max(tseries);
-  typename T::value_type range = (max - min);
-  ActsScalar medium = static_cast<ActsScalar>((max + min) * 0.5);
+  auto [minIt, maxIt] = std::ranges::minmax_element(tseries);
+  typename T::value_type range = (*maxIt - *minIt);
+  ActsScalar medium = static_cast<ActsScalar>((*maxIt + *minIt) * 0.5);
   return std::tie(range, medium);
 }
```

Comment:
Out of curiosity, the comment says:

```cpp
/// Return range and medium of a sorted numeric series
```

If the range is sorted, isn't the minimum always the first element, and isn't the maximum always the last element?

Reply:
Seems like the description is wrong.

In the tests we are not expecting it to be sorted:
```
  std::vector<ActsScalar> unordered = {-2., -1., 0., 1., 2., 3., -3.};
  auto [range1, medium1] = Acts::range_medium(unordered);
```

Reply:
Okay, in that case; could you please update the comment?